GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -102,19 +102,19 @@ private async void SendClassification(object obj)
             GetSubject();
         }
 
-        private bool CanSendClassification(object obj)
+        private bool CanSendClassification(object sender)
         {
             return CurrentAnnotation != null;
         }
 
-        private void ChooseAnswer(object obj)
+        private void ChooseAnswer(object sender)
         {
-            AnswerButton button = (AnswerButton)obj;
+            AnswerButton button = (AnswerButton)sender;
```

Comment:
Should there be something here to handle if the sender isn't castable to `AnswerButton`?

Reply:
I should be able to change this to `AnswerButton button= sender as AnswerButton;` to return `null` if uncastable rather than an exception.